### PR TITLE
Jail button in 'showreports'

### LINKED ISSUE #215: Have a jail and/or ban button on the report popup
Would be handy to have a button on the report popup to quickly pop someone into jail while investigating. Mockup image to show exactly what I mean.
jail button mockup

### PR DESCRIPTION
**Pull request by SimonFlapse**
**Jail button**
I tried to add a jail button to the 'showreports' gui.

I can't speak for the quality of the code.
I moved major parts of the jail_player(cmd) in custom_commands.lua to report.lua. Added a new button, downsized it and added an on_click event that calls report.jail(player, target)

I also added a notification sound for admins when a new report is made. (Or rather when the showreport windows is opened by a new report)

Feel free to improve it.

**What I haven't figured out:**
~~The strings "You have been placed in jail [...]" it prints out to the target seems to have an indentation.~~
~~^ also happens for the 'removed weapon' string~~
If it really works with multiple reports and different people.

Closes #215

### CODE REVIEW

File: report.lua [hunk 1]
```diff
@@ -93,6 +104,65 @@ function Module.report(reporting_player, reported_player, message)
     end
 end
 
+function Module.jail(player, target)
```
[the remark this answers is not in the document]
Comment (SimonFlapse) [reply report.lua]: That should do it. Sorry about that.

[the remark this answers is not in the document]
Comment (SimonFlapse) [reply report.lua]: It fixed the indentation on the console output. Thanks

File: report.lua [hunk 2]
```diff
@@ -6,8 +6,9 @@ local Game = require 'utils.game'
 local report_frame_name = Gui.uid_name()
 local report_close_button_name = Gui.uid_name()
 local report_tab_button_name = Gui.uid_name()
+local jail_offender_button_name = Gui.uid_name()
 local report_body_name = Gui.uid_name()
-
+local current_reported_player_name
```
[the remark this answers is not in the document]
Comment (SimonFlapse) [reply report.lua]: No desyncs for me

**Comment by SimonFlapse** [comment]
Causes conflict with #299

**Comment by iltar** [comment]
I agree, let's merge the refactor first. We know what we're testing and we know what breaks after the merge. It's a lot harder for the person refactoring this to test it afterwards.

**Pull request merged**